### DISCUSSION TITLE
Increasing the default token expiration time for 24 hours

### DISCUSSION THREAD

**Discussion by JSeam2** [Web3Auth/Web3Auth#631] [Help]
Is there a way of increasing the session token length on web3auth to go beyond 24 hours? Some users have provided feedback that they don't want to keep logging in everyday. Are there any best practices around this?

**Comment by rinzler99** [comment]
Hey, you can increase sesison time in web3auth sdk. Please refer below link:
https://web3auth.io/docs/sdk/web/web3auth/initialize#web3authoptions

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/30899350/191896884-456bcbd8-8e88-4aa2-bd63-93ffe806f20e.png">
